src/fixtures/node-module-next/function.js

```diff
@@ -0,0 +1,7 @@
+try {
+  // eslint-disable-next-line node/no-extraneous-require
+  require('next/dist/next-server/lib/constants')
+  // eslint-disable-next-line no-empty
+} catch (error) {}
```

Comment:
Do we need with `try`/`catch` block. I am assuming this file is read but never executed?

Reply:
No we don't, see https://github.com/netlify/zip-it-and-ship-it/pull/242/commits/61fc87169ec4c773840551b7ff670667f9dd102f